README.md

```diff
@@ -282,7 +282,7 @@ torchtune embodies PyTorch’s design philosophy [[details](https://pytorch.org/
 
 ### Native PyTorch
 
-torchtune is a native-PyTorch library. While we provide integrations with the surrounding ecosystem (eg: Hugging Face Datasets, EleutherAI Eval Harness), all of the core functionality is written in PyTorch.
+torchtune is a native-PyTorch library. While we provide integrations with the surrounding ecosystem (e.g: Hugging Face Datasets, EleutherAI Eval Harness), all of the core functionality is written in PyTorch.
```

Comment:
```suggestion
torchtune is a native-PyTorch library. While we provide integrations with the surrounding ecosystem (e.g. Hugging Face Datasets, EleutherAI Eval Harness), all of the core functionality is written in PyTorch.
```

Reply:
🤯🤯 💯💯 🔥🔥

Reply:
<img width="643" alt="Screenshot 2024-08-05 at 5 56 44 PM" src="https://github.com/user-attachments/assets/cbf808bb-6819-405e-9aa2-ab2f5cfda045">

TIL

Reply:
<img width="971" alt="image" src="https://github.com/user-attachments/assets/e6e50828-ed67-469a-b468-cb54b73ebfb2">

Reply:
slightly more esoteric;

<img width="971" alt="image" src="https://github.com/user-attachments/assets/57d235c2-f280-4558-8a73-e76ba2b59ab2">

Reply:
and who knew RIP was originally latin???
<img width="1131" alt="image" src="https://github.com/user-attachments/assets/8f3369ac-eb39-45ad-b466-e32f009e727b">

Reply:
Not reviewing anymore RFCs unless they end in QED so I know it was *demonstrated*